### PR TITLE
Dockertest Version 0.8.7 (NO API Changes)

### PR DESCRIPTION
**Pull request by cevich**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -723,6 +723,78 @@ Documentation Module
...
+
+#. Create a new branch off master with the next number:
+
+    ``git checkout -tb 0.8.7``
+
+#. Bump the version number up in three places:
+
+    ``$EDITOR config_defaults/defaults.ini dockertest/version.py conf.py``
+
+#. Creat a release commit:
```
Comment (edsantiago) [inline index.rst]: Creat**e**

File: index.rst [hunk 2]
```diff
@@ -723,6 +723,78 @@ Documentation Module
...
+
+    ``git commit -asm "Dockertest Version 0.8.5 (NO API Changes)"``
```
Comment (edsantiago) [inline index.rst]: You mean 0.8.**7** I assume

File: index.rst (hunk 1 above)
Comment (cevich) [inline index.rst]: thx.

File: index.rst (hunk 2 above)
Comment (cevich) [inline index.rst]: yep, thx.

File: index.rst [hunk 3]
```diff
@@ -723,6 +723,78 @@ Documentation Module
...
+   and due-date if desired.
+
+#. Move any currently open PRs the new milestone (right side pane of each PRs page).
+
+
+.. _`Create a new release`: https://github.com/autotest/autotest-docker/releases/new
+
+.. _`Create the next milestone`: https://github.com/autotest/autotest-docker/milestones/new
+
+.. _``close the previous milestone: https://github.com/autotest/autotest-docker/milestones
```
Comment (edsantiago) [inline index.rst]: Methinks you also need to fix this one

Comment (cevich) [inline index.rst]: oic, ````` in the wrong place.  kk got it.

**Pull request merged**